### PR TITLE
added FPWD to data-profile for "transwho"

### PR DESCRIPTION
**Pull request by koalie**
I want to make sure that for FPWD too the transition requests are sent to w3t-comm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -425,7 +425,7 @@ <h2 id="steps">Steps for
 <span class="transwho" data-profile="CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
 	creates a <a href="#transreq">transition request</a> in <a href='https://github.com/w3c/transitions/issues'>w3c/transitions</a>.
 </span>
-<span class="transwho" data-profile="REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
+<span class="transwho" data-profile="FPWD|REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
```
Comment (plehegar) [inline index.html]: Actually, we use GH for FPWD.
```suggestion
<span class="transwho" data-profile="REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
```

File: index.html [hunk 2]
```diff
@@ -425,7 +425,7 @@ <h2 id="steps">Steps for
 <span class="transwho" data-profile="CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
```
Comment (plehegar) [inline index.html]:
```suggestion
<span class="transwho" data-profile="FPWD|CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
```

This is already the case, eg. https://lists.w3.org/Archives/Team/w3t-comm/2020Jan/0137.html

**Comment by plehegar** [comment]
We should ask FPWD requests for go through the GH transitions repo. The notifier will then ensure it gets delivered to w3t-comm.

**Comment by plehegar** [comment]
For completion, w3t-comm appears at
 https://github.com/w3c/transition-issues-bot/blob/master/lib/notify-issue-transition.js#L61

**Comment by koalie** [comment]
Great!
Thanks, Philippe.

**Pull request closed without merging**